src/turbo_response/template.py

```diff
@@ -56,9 +59,15 @@ def render_turbo_frame_template(
 
     return render_turbo_frame(
         dom_id,
-        render_to_string(
-            template,
-            {**(context or {}), "turbo_frame_dom_id": dom_id, "is_turbo_frame": True},
-            **kwargs,
+        mark_safe(
```

Comment:
ditto

Reply:
Re Template: at module level you get the "apps not loaded yet" error. Presumably a workaround would be to load this library as a Django app, but that would break backward compatibility.  I could use lru_cache or similar for the time being.